core/src/test/java/org/testcontainers/containers/image/pull/policy/AgeBasedPullPolicyTest.java

```diff
@@ -0,0 +1,36 @@
+package org.testcontainers.containers.image.pull.policy;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Collections;
+import java.util.concurrent.TimeUnit;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.testcontainers.containers.image.ImageData;
+
+public class AgeBasedPullPolicyTest {
+
+    @Mock
+    private ImageData dockerImage;
+
+    @Before
+    public void init() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
if you make `ImageData` an interface, you can avoid mocks (and we prefer to avoid them where possible)

Reply:
It is a relatively "big" interface, so in this case it is easier to mock it than implement it just for the test, especially considering it has no business logic.